embedded/tools/stress_tool_sql/stress_tool_sql.go

```diff
@@ -153,7 +153,7 @@ func main() {
 		panic(err)
 	}
 
-	err = engine.SetDefaultDatabase("defaultdb")
+	_, _, err = engine.Exec("USE DATABASE defaultdb;", map[string]interface{}{}, nil)
```

Comment:
Minor: why not use `engine.SetCurrentDatabase("defaultdb")`?

Reply:
it was intentionally, to ensure everything can be done with SQL statements, but yes, if using `SetCurrentDatabase` programmatically some steps would be avoided